formats/pain.008.001.02.hbs

```diff
@@ -64,11 +64,16 @@
             <DtOfSgntr>{{mandateSignatureDate}}</DtOfSgntr>
             <AmdmntInd>false</AmdmntInd>
           </MndtRltdInf>
-        </DrctDbtTx>
-        {{#if bic}}
+        </DrctDbtTx>        
         <DbtrAgt>
             <FinInstnId>
+              {{#if bic}}
               <BIC>{{bic}}</BIC>
+              {{else}}
```

Comment:
This is a good change! My initial version only support BIC. But this is also allowed according the pain 008.001.02.xsd. And apparantly it is filled with NOTPROVIDED in that case.
Could be in some countries (Germany?) BIC is always expected though. But this will definitely work in the Netherlands e.g.

Reply:
In my case, yes, I am working out of Germany and collecting donations from bank accounts in germany. The BIC is only required on SEPA-Payments out of country, e.g. Netherlands.